src/components/StudyRoom/StudyRoomSettings.js

```diff
@@ -0,0 +1,183 @@
+import {useNavigate} from "react-router";
+import {useCallback, useEffect, useState} from "react";
+import * as React from "react";
+import axios from "axios";
+import {Avatar, InputLabel, Select, Stack, TextField} from "@mui/material";
+import MenuItem from "@mui/material/MenuItem";
+import CircleIcon from "@mui/icons-material/Circle";
+import {PrimaryButton3} from "../CustomMUIComponents/CustomButtons";
+import {PrimaryButton2} from "../CustomMUIComponents/CustomButtons";
+import MockUser from "../Profile/Mocks/mockUser.json"
+import MockStudyRoom from "./Mocks/mockStudyRoom.json"
+
+export default function StudyRoomSettings() {
+    const navigate = useNavigate();
+    const [errorMessage, setErrorMessage] = useState('')
+    const [roomData, setRoomData] = React.useState({
+        title:'',
+        color: '',
+        avatarText:'',
+        description:'',
+        participants:[],
+    });
+
+    const user = MockStudyRoom[1];
+
+    const fetchData = useCallback(() => {
+        axios.get('http://localhost:5000/room/{"username":"test"}')
+            .then(res => {
+                const data = res.data;
+                console.log(JSON.stringify(res.data));
+                setRoomData({...roomData, title: user.title});
+                setRoomData({...roomData, avatarText: user.avatarText});
+                setRoomData({...roomData, color: user.color});
+                setRoomData({...roomData, description: user.description});
+            })
+            .catch(err => {console.log(`Error: ${err}`); setErrorMessage(`${err}`.substring(44) === 401 ? 'request could not be sent' : `${err}`)});
+    },[])
+
+    React.useEffect(()=>{
+        let user = localStorage.getItem("username")
```

Comment:
email!

Reply:
I don;t have this on the branch im going to push again